### ISSUE TITLE
Cannot create new wallet - ("Wrong Captcha") bug? Please resolve.. its been over a month now

### ISSUE THREAD

**Issue by JustJackkie** [blockchain/service-my-wallet-v3#570]
I get this error when I try to create new wallet using api/v2/create ({"success":false,"message":"Wrong captcha"}).

Everything was working before then it just randomly started giving me this error out of nowhere. I am running node service and sending my requests to it - Service v0.26.3, Node v6.4.0

I have tried contacting blockchain regarding this issue for over a month now without a response.

Does anyone know how to resolve this?

**Comment by cryptoprogramer** [comment]
do you get to the new window which asks for new password

**Comment by Thore3** [comment]
man. Fuck blockchain!!!!!!!!! It’s unfortunately a bad player, witch will
be gone sooner than later I truly hope
14. mar. 2022 kl. 18:17 skrev JustJackkie ***@***.***>:

**Comment by Thore3** [comment]
I tell you!! Get as fast as you can out, and never look back

**Comment by aris2dev** [comment]
hi,

how did you get Api key? i tried many times but nobody replied me.

**Comment by cryptoprogramer** [comment]
@aris2dev 
if you get this error:
({"success":false,"message":"Wrong captcha"})
I can solve it

**Comment by aris2dev** [comment]
@isieditors 

No, i created success request but nobody replied me yet about API.

**Comment by mprenditore** [comment]
@isieditors can you please explain how to solve this?
I was able to create new wallets on August but now it's not working anymore.

Thanks

**Comment by cryptoprogramer** [comment]
@mprenditore 
what is your issue exactly?
 send me email or telegram message I hope i can help. you can find my contact details on my profile

**Comment by Thore3** [comment]
Wait till you tried a year!!
